src/Components/DemandArea.tsx

```diff
@@ -104,6 +106,7 @@ const DemandArea: React.FC<DemandAreaProps> = () => {
           label={demand.pizza.toString()}
           onIncrement={handleAddPizza}
           onDecrement={handleRemovePizza}
+          horizontal={true}
```

Comment:
This is equivalent to just passing `horizontal`

Reply:
Good point, I've changed it accordingly.